clients/src/test/java/org/apache/kafka/common/network/SslSelectorTest.java

```diff
@@ -67,7 +67,7 @@ public void setUp() throws Exception {
         this.channelBuilder = new SslChannelBuilder(Mode.CLIENT, null, false);
         this.channelBuilder.configure(sslClientConfigs);
         this.metrics = new Metrics();
-        this.selector = new Selector(5000, metrics, time, "MetricGroup", channelBuilder, new LogContext());
+        this.selector = new Selector(5000, 100, metrics, time, "MetricGroup", channelBuilder, new LogContext());
```

Comment:
Not sure why this was changed. This is the client-side selector. We are not delaying on clients? (Multiple occurrences)